### PR TITLE
Remove trailing zeroes

### PR DESCRIPTION
**Pull request by jessealama**
We remove support for trailing zeroes. This involves removing all discussion of cohorts and quanta, thereby considerably simplifying the data model.

### CODE REVIEW

File: spec.emu [hunk 1]
```diff
@@ -25,44 +25,41 @@ location: https://github.com/tc39/proposal-decimal/
...
     <emu-table id="table-decimal128-rounding-modes">
-      <emu-caption>Rounding modes in Decimal128 compared to IEEE 754 rounding modes</emu-caption>
+      <emu-caption>Rounding modes in Decimal128 compared to IEEE 754-2019 rounding modes</emu-caption>
       <table class="real-table">
         <thead>
           <tr>
             <th>Decimal128 rounding mode</th>
             <th>Description</th>
-            <th>IEEE 754 rounding mode</th>
+            <th>IEEE 754-2019rounding mode</th>
```
Comment (nicolo-ribaudo) [inline spec.emu:62]:
```suggestion
            <th>IEEE 754-2019 rounding mode</th>
```

File: intl.emu [hunk 2]
```diff
@@ -171,10 +170,8 @@
         <emu-alg>
           1. Let _p_ be _maxPrecision_.
           1. <ins>If _x_ is a Decimal128 value, then</ins>
-            1. <ins>Let _v_ be cohort(_x_).</ins>
-            1. <ins>Let _e_ be quantum(_x_).</ins>
-            1. <ins>Assert: _e_ ≤ 0.</ins>
-            1. <ins>Let _n_ be _v_ × 10<sup>-_e_</sup>.</ins>
+            1. <ins>Let _e_ be the smallest integer such that _x_ × 10<sup>-_e_</sup> is an integer.</ins>
```
Comment (nicolo-ribaudo) [inline intl.emu:173]: Can we have 0D here?

File: intl.emu [hunk 3]
```diff
@@ -299,11 +294,9 @@
           1. Else if _x_ is ~negative-infinity~, then
             1. Let _n_ be an ILD String value indicating negative infinity.
           1. <ins>Else if _x_ is a Decimal128 value, then</ins>
-            1. <ins>Let _v_ be cohort(_x_).</ins>
-            1. <ins>Let _q_ be quantum(_x_).</ins>
-            1. <ins>If _numberFormat_.[[Style]] is *"percent"*, set _x_ to « _v_ × 100, _q_ + 2 »<sub>𝔻</sub>.</ins>
+            1. <ins>If _numberFormat_.[[Style]] is *"percent"*, set _x_ to _x_ × 100.</ins>
             1. <ins>Set _exponent_ to the truncated exponent _x_.</ins>
-            1. <ins>Set _x_ to ! _v_ × 10<sup>-_exponent_</sup>.</ins>
+            1. <ins>Set _x_ to ! _x_ × 10<sup>-_exponent_</sup>.</ins>
```
Comment (nicolo-ribaudo) [inline intl.emu:299]: Also here, can we have "special" values?

File: intl.emu (hunk 2 above)
Comment (jessealama) [reply intl.emu:173]: > Can we have 0D here?

No, we can't. That's a good spot. I've moved up the special handling for 0 in the next if block and extended it to handle 0D, too.

File: intl.emu (hunk 3 above)
Comment (jessealama) [reply intl.emu]: We can't. Good catch. I've handled that now.

File: intl.emu [hunk 4]
```diff
@@ -87,7 +87,7 @@
         <h1>
           FormatNumericToString (
```
Comment (nicolo-ribaudo) [inline intl.emu:88]: You can delete all the AOs that now have no diff.

Comment (jessealama) [reply intl.emu]: Done

**Review by nicolo-ribaudo — APPROVED** [review]

**Pull request merged**